### DISCUSSION TITLE
How to auto reload on file save?

### DISCUSSION THREAD

**Discussion by KestutisMa** [Kitware/trame#335] [Q&A] [answered]
Hello,
While developing, after saving changes to app, there is manual repetitive actions of restarting trame app and then reloading web page.

I tried to enable "hot-reload" as stated here, but example is not reloading on save:
https://github.com/Kitware/trame/discussions/231
https://github.com/Kitware/trame/blob/master/examples/validation/core/17_hot_reload.py

Question: how to auto reload while developing app and after saving app file?

**Comment by jourdain** [comment, marked as answer]
https://kitware.github.io/trame/examples/core/basics.html#hot-reload

**Reply by KestutisMa** [reply]
Thank you
